mpas_analysis/ocean/time_series_ocean_regions.py

```diff
@@ -433,20 +433,22 @@ def run_task(self):  # {{{
                     units = timeSeries.units
                     description = timeSeries.long_name
 
-                    if 'nVertLevels' in timeSeries.dims:
+                    is3d = 'nVertLevels' in timeSeries.dims
+                    if is3d:
                         timeSeries = \
                             (dsRegion.volCell*timeSeries.where(depthMask)).sum(
                                 dim='nVertLevels').sum(dim='nCells') / totalVol
                     else:
                         timeSeries = \
-                            (dsRegion.areaCell*timeSeries.where(cellMask)).sum(
+                            (dsRegion.areaCell*timeSeries).sum(
                                 dim='nCells') / totalArea
 
                     timeSeries.compute()
 
                     dsOut[outName] = timeSeries
                     dsOut[outName].attrs['units'] = units
                     dsOut[outName].attrs['description'] = description
+                    dsOut[outName].attrs['is3d'] = str(is3d)
```

Comment:
Here, I store the attribute as a string because I'm not sure a bool would work.  Maybe that's okay in NetCDF?

Reply:
yeah, I don't know. seems safer this way.